test-errors
testMasterExceptionFailsTest
	<expectedFailure>
	
	self localDo: [ self error ]